### DISCUSSION TITLE
kV8IntelJCCErratumMitigation v8 flag

### DISCUSSION THREAD

**Discussion by uazo** [Alex313031/Thorium-Win#196] [Show and tell]
Ciao @Alex313031

I was checking the new chromium flags and found something that might interest thorium.

Issue 14225: Implement JCC erratum performance mitigation in TurboFan
https://bugs.chromium.org/p/v8/issues/detail?id=14225
https://www.intel.com/content/dam/support/us/en/documents/processors/mitigations-jump-conditional-code-erratum.pdf

maybe you don't need it or you already knew it, I don't understand anything about these things!

**Comment by gz83** [comment]
Close discussions that are outdated and inactive.
